scipy-1.0/paper.tex

```diff
@@ -902,6 +902,53 @@ \subsection*{Maintainers and contributors}
 are selected on a rolling basis from contributors with a significant history of
 high-quality contributions.
 
+\subsection*{Funding}
+
+
+
+%100,000 annual salary, 112 person-years, 414,473 lines of code -> 11,192,622 (https://www.openhub.net/p/scipy/estimated\_cost, which uses the Basic COCOMO model).
+
+The development cost of SciPy is estimated in excess of 10M dollars by
+OpenHub.  Yet, the project is largely unfunded, having been
+developed predominantly by graduate students, faculty, and members of industry
+in their free time. Small amounts of funding have been applied with success:
+some meetings were sponsored by universities and industry, Google's Summer of Code
+program supported infrastructure and algorithmic work, and teaching grant funds were
+used early on to develop documentation.  However, these small amounts of
+sporadic funding are not commensurate with the enormous stack of important scientific
+software, like those that are used to plan national space missions, that rely
+on SciPy.  In the traditional academic funding model, it is easy to support
+commercial software, but hard to provide funds for community driven projects,
+and as more science depends on open source software national funding agencies
+urgently need to pay attention to this challenge. And, unlike for commercial
+software where a license seat encompasses all development (and other) costs, agencies will
+have to tolerate the more diverse spending required to do planning and maintain
```

Comment:
Does this mean that it is a very clear line item when funding seats of commercial software, and it is not so easy to write on paper what the costs are for open source projects?

Reply:
Not so much that it is hard to write down the costs as it is hard to get funding for them.  For instance, NIH commonly provides funding to support matlab seat licenses for neuroimaging research, but has only provided minimal funds for numpy, scipy, matplotlib, etc.  I don't have hard numbers, but I suspect the US tax payers have spent significantly more on matlab than on the scientific Python stack.  There are various reasons for that, but I think we only need to mention the fact here.